packages/vuetify/src/services/icons/presets/index.ts

```diff
@@ -1,11 +1,16 @@
 // Types
 import { VuetifyIconSets } from 'vuetify/types/services/icons'
 
+import mdiSvg from './mdi-svg'
```

Comment:
There is no relation between that and mdi. It works with any svg path. Imho it should be `svg`.

Reply:
But it is mdi's svg path that provides us with these values so it will stand as such.